src/charm.py

```diff
@@ -666,12 +666,15 @@ def _create_upf_configuration_file(self) -> tuple[bool, bool]:
         restart = False
         recreate_pod = False
         core_ip_address = self._get_network_ip_config(CORE_INTERFACE_NAME)
+        core_ip_masquerade_flag = self._charm_config.core_ip_masquerade
```

Comment:
```suggestion
        core_ip_masquerade_enabled = self._charm_config.core_ip_masquerade
```

I think this would a bit clearer with this name. Maybe the charm configuration itself could be renamed like that, WDYT?

Reply:
Agreed, fixed